code/game/objects/structures.dm

```diff
@@ -64,6 +64,11 @@
 		step(O, get_dir(O, src))
 	return
 
+/obj/structure/proc/do_climb(atom/movable/A)
+	density = 0

```

Comment:
This is just a refactor of /obj/structures climb code so forced_movement can use it. Should be fine if it was before